sunbeamlib/__init__.py

```diff
@@ -90,3 +91,13 @@ def circular(seq, kmin, kmax, min_len):
             return True
     return False
 
+def read_seq_ids(fasta_fp):
```

Comment:
OK, I think the new commits should take care of both of these things.  (It marked the bowtie.rules feedback as outdated, but not this one.  Is that because that marked line in `__init__.py` is unchanged?  90f755f shows the actual change, in any case.)